src/Farmer/Arm/Network.fs

```diff
@@ -621,6 +621,7 @@ type NetworkInterface =
 
 
 
+
```

Comment:
remove?

Reply:
fantomas probably did this too, but i'll check